tests/migration/migration.go

```diff
@@ -2725,7 +2725,12 @@ var _ = SIGMigrationDescribe("VM Live Migration", decorators.RequiresTwoSchedula
 
 			By("Creating ResourceQuota with enough memory for the vmi but not enough for migration")
 			resourceQuota := newResourceQuota(resourcesToLimit, testsuite.GetTestNamespace(vmi))
-			_ = createResourceQuota(resourceQuota)
+			resourceQuota = createResourceQuota(resourceQuota)
+			Eventually(func(g gomegatypes.Gomega) error {
+				quota, err := virtClient.CoreV1().ResourceQuotas(resourceQuota.Namespace).Get(context.TODO(), resourceQuota.Name, metav1.GetOptions{})
+				g.Expect(quota.Status.Hard).To(Equal(resourcesToLimit))
```

Comment:
/lgtm cancel


I think that you need to you use `func (q *Quantity) Cmp(y Quantity) int`
please consider using Quota.Status.Used i think that there was a good reason that kubernetes used this field